README.rst

```diff
@@ -24,3 +24,16 @@ If you are interested in contributing or extending the project, take a look at t
 .. |Requirements Status| image:: https://requires.io/github/openoereb/pyramid_oereb/requirements.svg?branch=master
    :target: https://requires.io/github/openoereb/pyramid_oereb/requirements/?branch=master
    :alt: Requirements Status
+
+tl;nr - Local testing
+---------------------
+
+Run:
+
+.. code-block:: bash
+
+  git clone git@github.com:openoereb/pyramid_oereb.git
+  cd pyramid_oereb
+  make serve
+
+JSON embeddable extract is accessible at: http://localhost:6543/oereb/extract/embeddable/json/CH113928077734.
```

Comment:
better to use the reduced flavour as "normal" testing flavour, in my opinion

Reply:
Done